### PR TITLE
Implement object interface for namespaces and volumes

### PR DESCRIPTION
**Pull request by croomes**
Templates to add the Object interface methods to namespace and volume models.

### CODE REVIEW

File: v2/model_volume.go [hunk 1]
```diff
@@ -36,3 +36,23 @@ type Volume struct {
...
+	return t.Id
+}
+
+// GetName returns the volume name.
+func (t Volume) GetName() string {
+	return t.Name
+}
+
+// GetNamespace returns the volume namespace. 
+func (t Volume) GetNamespace() string {
```
Comment (avestuk) [inline v2/model_volume.go:51]: Should this not be returning the volume namespace?

File: v2/templates/volume/model.mustache [hunk 2]
```diff
@@ -39,18 +39,23 @@ type {{classname}} struct {
...
+// GetName returns the volume name.
+func (t {{classname}}) GetName() string {
 	return t.Name
 }
 
-// GetLabels returns the object's labels.
-func (t *{{classname}}) GetLabels() map[string]string {
+// GetNamespace returns the volume namespace. 
+func (t {{classname}}) GetNamespace() string {
+	return ""
```
Comment (avestuk) [inline v2/templates/volume/model.mustache:54]:
```suggestion
	return t.NamespaceID
```

File: v2/model_volume.go (hunk 1 above)
Comment (croomes) [reply v2/model_volume.go:51]: Thanks!

**Review by darkowlzz — APPROVED** [review]
LGTM!

**Review by avestuk — APPROVED** [review]

**Pull request merged**